src/screens/UserPortal/People/People.spec.tsx

```
@@ -34,157 +29,360 @@ import { vi } from 'vitest';
+          })),
+          pageInfo: {
+            endCursor: 'cursor5',
+            hasPreviousPage: false,
+            hasNextPage: true,
+            startCursor: 'cursor1',
+          },
+        },
+      },
+    },
+  },
 };
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)